oncrpc/nt.c

```diff
@@ -33,11 +33,11 @@ int rpc_nt_init(void)
     if (init++)
 	return 0;
 	
-    if (WSAStartup(0x0101, &WSAData)) {
-	init = 0;
-	nt_rpc_report("WSAStartup failed\n");
-	WSACleanup();
-	return -1;
+    if (WSAStartup(MAKEWORD(2,2), &WSAData)) {
```

Comment:
Why is `0x0101` changed to `MAKEWORD(2, 2)`? The result of `MAKEWORD(2, 2)` is `0x0202`!
Shouldn't this be `MAKEWORD(1, 1)`?

Reply:
The `0x0101` represent the version of Win socket to use. This pull request updates to use the newest standard (2,2) and using `MAKEWORD` is the preferred way of creating this version number.

https://docs.microsoft.com/en-us/windows/win32/api/winsock/nf-winsock-wsastartup

Reply:
Ok! Thanks for the clarification.